x-pack/plugins/lens/public/visualizations/gauge/visualization.tsx

```diff
@@ -222,6 +223,8 @@ export const getGaugeVisualization = ({
   },
   getSuggestions,
 
+  triggers: [VIS_EVENT_TO_TRIGGER.filter],
```

Comment:
Tested and realized gauges are never dispatching filter events (you can't click a gauge). Sorry for putting you on the wrong track here.